db/migrate/20160405143543_create_git_references.rb

```diff
@@ -0,0 +1,14 @@
+class CreateGitReferences < ActiveRecord::Migration[5.0]
+  def change
+    create_table :git_references do |t|
+      t.string    :name
+      t.string    :commit_sha
+      t.timestamp :commit_time
+      t.text      :commit_message
+      t.string    :type
+      t.bigint    :git_repository_id
+
+      t.timestamps
```

Comment:
I think you need `:null => false` or something here or Rails emits a warning on db:migrate...can't remember though (@jrafanie?)